### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by 1478599553**

### CODE REVIEW

File: src/main/resources/assets/pyrotech/lang/zh_cn.lang [hunk 1]
```diff
@@ -350,16 +352,18 @@ gui.pyrotech.tooltip.torch.burns.false=持续燃烧。
 gui.pyrotech.tooltip.rain.extinguishes.true=淋雨后熄灭，
 gui.pyrotech.tooltip.rain.extinguishes.false=淋雨后仍燃烧。
 gui.pyrotech.tooltip.potency=Potency: %d%%
```
Comment (codetaylor) [inline src/main/resources/assets/pyrotech/lang/zh_cn.lang:354]: The potency tooltip describes the magnitude of the effect that eating a Roasted Marshmallow has.

File: src/main/resources/assets/pyrotech/lang/zh_cn.lang [hunk 2]
```diff
@@ -350,16 +352,18 @@ gui.pyrotech.tooltip.torch.burns.false=持续燃烧。
...
-gui.pyrotech.tooltip.redstone.active.durability=Has a %s%d%%%s chance to ignore damage while active.
-gui.pyrotech.tooltip.redstone.active.efficiency=Breaks blocks %s%d%%%s faster while active.
-gui.pyrotech.tooltip.redstone.active.damage=Deals %s%d%%%s damage while active.
-gui.pyrotech.tooltip.redstone.active.hoe=Tills a %s3x3%s area while active.
-gui.pyrotech.tooltip.compost.value=Compost Value: %d
-
-gui.pyrotech.tooltip.quartz.active.efficiency=Breaks blocks %s%d%%%s faster in %sThe Nether%s.
-gui.pyrotech.tooltip.quartz.active.damage=Deals %s%d%%%s damage in %sThe Nether%s.
-gui.pyrotech.tooltip.quartz.active.hoe=Tills a %s3x3%s area in %sThe Nether%s.
+#I (DRAMING)couldn't find this entry, so I didn't translate it.(Could you please tell me?)
```
Comment (codetaylor) [inline src/main/resources/assets/pyrotech/lang/zh_cn.lang:355]: The potency tooltip describes the magnitude of the effect that eating a Roasted Marshmallow has.

File: src/main/resources/assets/pyrotech/lang/zh_cn.lang [hunk 3]
```diff
@@ -327,13 +327,15 @@ gui.pyrotech.tooltip.uses.full=耐久：%d
 gui.pyrotech.tooltip.durability=耐久：%d / %d
 gui.pyrotech.tooltip.durability.full=耐久：%d
 gui.pyrotech.tooltip.durability.indestructible=Indestructible
+#I (DRAMING)couldn't find this entry, so I didn't translate it.(Could you please tell me?)
```
Comment (codetaylor) [inline src/main/resources/assets/pyrotech/lang/zh_cn.lang:330]: The indestructible tooltip is used when a cog or sawblade is flagged as indestructible in the config file.

Comment (1478599553) [reply src/main/resources/assets/pyrotech/lang/zh_cn.lang:330]: Thanks!

File: src/main/resources/assets/pyrotech/lang/zh_cn.lang (hunk 2 above)
Comment (1478599553) [reply src/main/resources/assets/pyrotech/lang/zh_cn.lang:353]: But how can I roast a marshmallow? I couldn't find any recipes in the Jei.(I do not really play Pyrotech well, lol)

Comment (codetaylor) [reply src/main/resources/assets/pyrotech/lang/zh_cn.lang]: Craft a stick with a sword or axe to get a sharpened stick. Place sharpened stick in main hand and marshmallow in offhand and click to combine. Stand near campfire and use combined stick / marshmallow on the campfire. Be careful not to burn it.

Comment (1478599553) [reply src/main/resources/assets/pyrotech/lang/zh_cn.lang]: Okay. I see. Thanks a lot!!

**Comment by codetaylor** [comment]
Thank you for the translation.

Do you plan to finish the translation?

**Comment by 1478599553** [comment]
Oh , sorry.I was busy the past days. So I will keep on translating from now on.

**Comment by codetaylor** [comment]
Great, thank you! Let me know when it's finished.

File: src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/survive/campfire_effects.json [hunk 4]
```diff
@@ -1,45 +1,45 @@
...
       "image": "pyrotech:textures/potions/comfort.png",
-      "text": "Stand within range of a campfire at night to gain the $(thing)Comfortable$() effect.$(p)$(info) The $(thing)Comfortable$() effect increases the effect of eating.$(p)$(warning) This effect will disappear if hostile mobs are nearby."
+      "text": "在晚上时，站在篝火旁一定范围来获得$(thing)舒适$()效果。$(p)$(info)$(thing)舒适$()效果使进食的回复效果更好。$(p)$(warning)如果附近有怪物在游荡，这个效果就会消失。"
     },
     {
       "type": "campfire_effect",
-      "title": "Well Fed",
+      "title": "吃饱喝足",
       "image": "pyrotech:textures/potions/well_fed.png",
-      "text": "Eat a lot of food with the $(thing)Comfortable$() effect to gain the $(thing)Well Fed$() effect.$(p)$(info) The $(thing)Well Fed$() effect increases your stamina.$(p)$(warning) This effect will disappear over time."
```
Comment (1478599553) [inline src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/survive/campfire_effects.json:24]: what does the "stamina" means?

Comment (codetaylor) [reply src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/survive/campfire_effects.json:24]: In this context stamina refers to the rate at which a player becomes hungry while performing tasks that reduce their hunger haunches. When the player has the `Well Fed` effect, the rate they become hungry is reduced and it takes longer for them to get hungry while performing tasks.

**Comment by 1478599553** [comment]
It is 90% finished now, I think. But I'm going to school now, and I couldn't keep translating. So just please merge it, I'll complete it when I go back home.

**Comment by EnterFor** [comment]
@codetaylor Can you temporarily merge it? I can't wait to play !!!!  ღ( ´･ᴗ･` )

**Comment by codetaylor** [comment]
I prefer to merge it only once after it is complete and I have had a chance to review it. Thank you for your patience.

File: src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/bloom/tools_redstone.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "text",
-      "text": "$(info) $(thing)红石工具$()会在耐久消耗或待在$(thing)红石矿石$()附近时被激活。$(p)$(info) $(thing)红石工具$()会在玩家待在$(thing)红石矿石$()附近时缓慢回复耐久。$(p)$(info) Active $(thing)Redstone Tools$() resist damage and are more likely to remain active by taking damage."
+      "text": "$(info) $(thing)红石工具$()会在耐久消耗或待在$(thing)红石矿石$()附近时被激活。$(p)$(info) $(thing)红石工具$()会在玩家待在$(thing)红石矿石$()附近时缓慢回复耐久。$(p)$(info) 激活的$(thing)红石工具$() resist damage and are more likely to remain active by taking damage."
```
Comment (1478599553) [inline src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/bloom/tools_redstone.json:8]: What does "resist damage" means?
Won't the tools take damage? Or just take damage hardly?

Comment (codetaylor) [reply src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/bloom/tools_redstone.json:8]: It means that there is less of a chance that they will take damage when used.

Comment (1478599553) [reply src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/bloom/tools_redstone.json:8]: okay, thanks

**Comment by 1478599553** [comment]
Okaaaaaaaaaaaay!
I complete it totally.
Please merge this PR.
Thanks.

**Comment by codetaylor** [comment]
Thank you.
It looks good.
I will merge it in the next few days.

**Pull request merged**